packages/slice-machine/lib/builders/common/Zone/components/EmptyState/index.tsx

```diff
@@ -13,7 +13,7 @@ const ZoneEmptyState = ({
     <Heading as="h5">Add a new field here</Heading>
     <Box sx={{ my: 2 }}>
       <Text sx={{ color: "textClear" }}>
-        Add a field to your {zoneName} Zone
+        Fields hold content, like text and images.
```

Comment:
_✍️ potential change_

After testing your work, I feel like we could be a bit more explicit about the fact there is a field type for repeatable content. 
Like `Fields hold content, like text, images, repeatable group of fields.`
WDYT?

Reply:
I don't think we need to advertise the types of fields too much. "Group of fields" isn't really content; it's more of a content structure, so it sounds misplaced in this sentence.

@samlfair recommended this text:
> Fields are inputs for content (e.g. text, images, links)

I think that works well. Developers will quickly see that they can add groups once they click "Add a field".

What do you think? Should we open this up to the rest of the team?

Reply:
Let's do that, no need 🙂